Xamarin.Forms.Platform.UAP/TabbedPageRenderer.cs

```diff
@@ -543,5 +551,32 @@ void SetAppBarForegroundBinding(FrameworkElement element)
 				new Windows.UI.Xaml.Data.Binding { Path = new PropertyPath("ToolbarForeground"), 
 					Source = Control, RelativeSource = new RelativeSource { Mode = RelativeSourceMode.TemplatedParent } });
 		}
+
+		void UpdateSelectedTabColors()
+		{
+			foreach (WTextBlock tabBarTextBlock in Control.GetDescendantsByName<WTextBlock>(TabBarHeaderTextBlockName))
+			{
+				if (tabBarTextBlock.Text == Element.CurrentPage.Title)
```

Comment:
What happens if you have multiple tabs with the same title?

Reply:
Updated with code that makes more sense :)